pkgs/development/python-modules/proton-vpn-killswitch-network-manager-wireguard/default.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  gobject-introspection,
+  setuptools,
+  networkmanager,
+  proton-vpn-api-core,
+  proton-vpn-killswitch,
+  proton-vpn-logger,
+  pycairo,
+  pygobject3,
+  pytestCheckHook,
+  iproute2,
+  nix-update-script,
+}:
+
+buildPythonPackage rec {
+  pname = "proton-vpn-killswitch-network-manager-wireguard";
+  version = "0.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ProtonVPN";
+    repo = "python-proton-vpn-killswitch-network-manager-wireguard";
+    rev = "v${version}";
+    hash = "sha256-Ftqab+OVh7rK9l1A3KmP7gXLztiCaayB16cmqTTou4g=";
+  };
+
+  nativeBuildInputs = [
+    # Solves ImportError: cannot import name NM, introspection typelib not found
+    gobject-introspection
+    setuptools
+  ];
+
+  propagatedBuildInputs = [
```

Comment:
```suggestion
  dependencies = [
```

Reply:
See below - can do it for all of protonvpn-gui's packages in this MR as one additional commit or as follow-up PR